Tasks/DownloadPipelineArtifactV1/task.loc.json

```diff
@@ -9,12 +9,12 @@
   "version": {
     "Major": 1,
     "Minor": 1,
-    "Patch": 2
+    "Patch": 1
```

Comment:
Why is this going down?

Reply:
This was changed by mistake by this PR : https://github.com/Microsoft/azure-pipelines-tasks/pull/10139/files, but we didn't change the agent plugin version, hence reverting.